Files/Strings/ja-JP/Resources.resw

```diff
@@ -910,13 +910,13 @@
     <value>ターミナルアプリ</value>
   </data>
   <data name="SettingsAboutSupportUs.Text" xml:space="preserve">
-    <value>応援する</value>
+    <value>支援する</value>
   </data>
   <data name="SettingsAboutSupportUsDescription.Text" xml:space="preserve">
-    <value>PayPalで応援する</value>
+    <value>PayPalで支援する</value>
```

Comment:
Maybe:
```suggestion
    <value>PayPal で支援する</value>
```

As you did at: https://github.com/files-community/Files/pull/2848/files#diff-5bce934836368afdd400bfe79c43711d851b1137c2ec59d51b1b3ea1ca8e717eR259

Reply:
Thank you! I applied.